messages.proto

```diff
@@ -92,6 +92,18 @@ enum MessageType {
 	MessageType_EthereumMessageSignature = 110 [(wire_out) = true];
 	MessageType_ChangeWipeCode = 111 [(wire_in) = true];
 
+	// eip-712 messages
+  MessageType_EthereumSignTypedHash = 112 [(wire_in) = true];
+  MessageType_EthereumTypedDataSignature = 113 [(wire_out) = true];
+
+	// not implemented
+  //MessageType_EthereumSignTypedData = 114 [(wire_in) = true];
+  //MessageType_EthereumTypedDataStructRequest = 115 [(wire_out) = true];
+  //MessageType_EthereumTypedDataStructAck = 116 [(wire_in) = true];
+  //MessageType_EthereumTypedDataValueRequest = 117 [(wire_out) = true];
+  //MessageType_EthereumTypedDataValueAck = 118 [(wire_in) = true];
```

Comment:
Even though support for these messages is currently unimplemented, is there any issue with adding these definitions to the device-protocol library now?

Reply:
Those are the messages Trezor uses which we will not be using. I will delete these.